src/samVisualize/rendering/icons.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { readdirSync } from 'fs'
+import { join } from 'path'
+import * as vscode from 'vscode'
+/**
+ * Maps a resource type to a Webview URI pointing to a corresponding svg icon
+ */
+export type IconURIMap = {
+    [index: string]: string
```

Comment:
What's `index`?

Reply:
https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures

Reply:
Sure, that’ll be clearer